### ISSUE TITLE
Seeded random bias for sensors

### ISSUE THREAD

**Issue by iandareid** [rosflight/rosflight_ros_pkgs#167]
The sensors having random bias on each start means that it must be calibrated on each launch. This has issues because of the collision detection that Gazebo uses, since the aircraft bounces on the plane it rests on. We should have a seeded random bias so that we can calibrate only once when installing ROSplane.

**Comment by iandareid** [comment]
Duplicate of issue #168.
